packages/utilities/tests/objectValues.test.ts

```diff
@@ -0,0 +1,17 @@
+import { objectValues } from '../src';
+
+describe('objectValues', () => {
+	test('GIVEN basic THEN returns expected', () => {
+		const source = { a: 'Hello', b: 420 };
+		const expected = ['Hello', 420];
+
+		expect(objectValues(source)).toEqual(expected);
```

Comment:
You should at very least type assert the `expect()` call, I believe this would be `expect<string | number>(source)`.

If you do `as const` (please test that readonly works!), it would be `expect<'Hello' | 420>(source)` for enhanced safety.

Reply:
```typescript
	test('GIVEN basic THEN returns expected', () => {
		const source = { a: 'Hello', b: 420 } as const;
		const expected = ['Hello', 420] as const;

		expect<('Hello' | 420)[]>(objectValues(source)).toEqual(expected);
	});
```
Like so?

Reply:
Yes, although `expected` doesn't need `as const`, and we should have a type without readonly types.